_posts/en/newsletters/2020-06-17-newsletter.md

```diff
@@ -0,0 +1,193 @@
+      funded which outputs.
+
+        Because the coordinator is unable to view the output at the time
+        it creates its blinded signature, it can't allow the user to
+        specify an arbitrary amount or the user could attempt to receive
+        more money than they contributed to the coinjoin.  This isn't a
+        security risk---the other participants will refuse to sign any
+        malformed transaction---but such a failure requires restarting
+        the protocol.  If arbitrary amounts were allowed, the blinding
+        would prevent identification of the lying user and make it
+        impossible to ban them from future rounds, allowing an unlimited
+        DoS of the protocol.  Instead, Wasabi requires all outputs
+        either belong to a small set of allowed sizes (e.g. 0.1 BTC, 0.2
+        BTC, 0.4 BTC, etc) or be an unblinded change output.  This
+        limits the ability to use Wasabi with user-specified amounts or
+        for making payments with arbitrary amounts.
+
+    This week, several contributors to Wasabi [posted][wabisabi post]
+    to the Bitcoin-Dev mailing list about a new
+    protocol they call WabiSabi that conceptually extends their existing protocol
+    with a technique adapted from [confidential transactions][].  This allows a
+    client to create a commitment to arbitrary output amounts and---without
+    revealing the amounts---prove that each amount is individually
+    within a specified range (e.g. 0.0001 BTC to 21 million BTC) and
+    that they collectively sum to a specified value.  The coordinator
+    uses this specified value to verify that the sum of the outputs the
+    client wants to create is equal to the sum of the inputs provided by
+    the client (minus fees).  The coordinator can then provide an
+    anonymous credential for each output commitment that allows the
+    client to later anonymously submit the revealed output to the
+    coordinator for inclusion in the transaction template.
+
+    Software that implements the protocol will be able to create
+    coordinated coinjoins---with privacy from the coordinator---that
+    allow the clients to select their output amounts, facilitating
+    experiments with non-equal value coinjoins (see [Newsletter
+    #79][news79 unequal coinjoin]) and payments made within a coinjoin
+    (either to third-parties not participating in the coinjoin or to
+    parties within a coinjoin).
+
+    The proposed protocol contains a significant number of differences
+    from Wasabi's current protocol (such as replacing blind signatures
+    with keyed-verification anonymous credentials), so its authors are
+    seeking review, criticism, and suggestions about how the protocol
+    can be used most effectively.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Caravan adds BIP32, coin selection, hardware wallet test suite:**
```

Comment:
I think this is coin *control* not coin *selection*.  See: https://twitter.com/murchandamus/status/1256270340008730624

Reply:
Nice catch!